p11-kit/test-filter.c

```diff
@@ -138,10 +138,10 @@ test_allowed (void)
 	rv = (module->C_GetMechanismInfo) (slots[0], mechs[0], &mech);
 	assert_num_eq (CKR_OK, rv);
 
-	rv = (module->C_InitToken) (99, (CK_UTF8CHAR_PTR)"TEST PIN", 8, (CK_UTF8CHAR_PTR)"TEST LABEL");
+	rv = (module->C_InitToken) (99, (CK_UTF8CHAR_PTR)"TEST PIN", 8, (CK_UTF8CHAR_PTR)"TEST LABEL                      ");
```

Comment:
Should we guard this with:
```c
#if P11_RPC_PROTOCOL_VERSION_MAXIMUM == 1
...
#endif
```

Reply:
it shouldnt be necessary as the strings are both NULL terminated and space padded